pyanaconda/nm.py

```diff
@@ -751,7 +751,11 @@ def _find_settings(value, key1, key2, format_value=lambda x:x):
     connections = proxy.ListConnections()
     for con in connections:
         proxy = _get_proxy(object_path=con, interface_name="org.freedesktop.NetworkManager.Settings.Connection")
-        settings = proxy.GetSettings()
+        try:
+            settings = proxy.GetSettings()
+        except GLib.GError as e:
+            log.debug("_find_settings: %s", e)
```

Comment:
For a debug message, to me it doesn't make much difference, but I can change it.